lang/java/ipc/src/main/java/org/apache/avro/ipc/HttpServer.java

```diff
@@ -48,38 +50,47 @@ public HttpServer(Responder responder, String bindAddress, int port) throws IOEx
 
   /** Constructs a server to run on the named port on the specified address. */
   public HttpServer(ResponderServlet servlet, String bindAddress, int port) throws IOException {
-    this.server = new org.mortbay.jetty.Server();
-    SelectChannelConnector connector = new SelectChannelConnector();
-    connector.setLowResourceMaxIdleTime(10000);
-    connector.setAcceptQueueSize(128);
```

Comment:
Would it make sense to set this, as well as setIdleTime so it's more similar to the previous version?